beast/tools/run/create_physicsmodel.py

```diff
@@ -155,7 +156,7 @@ def create_physicsmodel(nsubs=1, nprocs=1, subset=[None, None]):
 
         # function to process the subgrids individually
         def gen_subgrid(i, sub_name):
-            sub_g_pspec = FileSEDGrid(sub_name)
+            sub_g_pspec = SEDGrid(sub_name)
```

Comment:
This should be `SpectralGrid(sub_name)` instead.

Reply:
Done.